libs/tinygettext/src/language.cpp

```diff
@@ -281,6 +281,9 @@ static const LanguageSpec languages[] = {
   { "zh", "CN", nullptr, "Chinese (simplified)"        },
   { "zh", "HK", nullptr, "Chinese (Hong Kong)"         },
   { "zh", "TW", nullptr, "Chinese (traditional)"       },
+  { "zh", "Hans",     nullptr, "Chinese (simplified)"        },
+  { "zh", "Hant_HK",  nullptr, "Chinese (Hong Kong)"         },
+  { "zh", "Hant",     nullptr, "Chinese (traditional)"       },
```

Comment:
Isn't it confusing if the “long name” (in parenthesis) is the same as other values? Can those lang code be fully exchanged? Is zh_Hant always usable instead of zh_TW as a code for the same translation (and the reverse being true), basically zh_TW being an alias to zh_Hant?

Reply:
For `zh_Hans` the `Han` represents 「漢」“Chinese” and `s` for “simplified”. I think it is the `zh_CN` to be changed to `Chinese (China)` instead, because the `CN` part really means a country.

Which one to use depends on our choices of infrastructure (as the situation stated in #2938), and we only choose one between `zh_Hans` and `zh_CN`, as their difference can be disregarded after then.

I’d differentiate them as you said.

Reply:
Thanks.